spring-cloud-gcp-starters/spring-cloud-gcp-starter-metrics/.repo-metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "spring-cloud-gcp-starter-metrics",
```

Comment:
What's this file for? I guess we missed discussing it in the [original PR](https://github.com/spring-cloud/spring-cloud-gcp/pull/2391).
We don't do this anywhere else. I wonder how this is used.

Reply:
This is OK; I added .repo-metadata.json for all of our starters so that the information is picked up by DRIFT APIs and our library data is accessible by Cloud Code.